aarch64/lib/config_default.toml

```diff
@@ -0,0 +1,59 @@
+[build]
+target = "lib/aarch64-unknown-none-elf.json"
+buildflags = [
+    "-Z", "build-std=core,alloc"
+]
+
+[link]
+arch = 'aarch64'
+script = 'aarch64/lib/kernel.ld'
+
+# This needs to change to a virtual address once we set up paging.
+load-address = '0xffff800000100000 - 0x80000'
+
+[config]
+platform = "virt"
+features = []
+
+dev = [

```

Comment:
could we remove out the sections that aren't currently used?

you could add an explanation of how it can be used in certain circumstances instead.

Reply:
sure, this is possible